CONTRIBUTING.md

```diff
@@ -4,8 +4,11 @@ Thanks for taking your time to contribute!
 
 First off, any and all contributions are welcome.
 
-Secondly, if your pull request or issue is in any way related to the [AUR package](https://aur.archlinux.org/packages/betterlockscreen-git/), 
-for example if your pull request adds a new dependency, 
-please notify the maintainer @AUTplayed via a mention (like used here with the @) in your comment.
+Secondly, if your pull request or issue is in any way related to the [AUR package](https://aur.archlinux.org/packages/betterlockscreen-git/),
+please notify the maintainer @Maik93 via a mention (like used here with the @) in your comment.
+
+Also, in case of adding new dependencies, please notify @Maik93 and @m1m3-50.
```

Comment:
Hey @m1m3-50, do you still want to get active notifications for dependency changes in Betterlockscreen? Didn't find something about this :)

Reply:
If either is unable to maintain the AUR packages, I can take them over.